inference-engine/src/auto_plugin/auto_plugin.cpp

```diff
@@ -179,7 +330,26 @@ std::vector<AutoPlugin::DeviceInformation> AutoInferencePlugin::GetDeviceChoice(
     return metaDevices;
 }
 
-//////////////////////////////////// private & protected functions ///////////////////
+std::vector<std::string> AutoInferencePlugin::GetOptimizationCapabilities() const {
+    // FIXME: workaround to get devicelist.
+    std::unordered_set<std::string> capabilities;
+    std::vector<std::string> queryDeviceLists{"CPU", "GPU"};
+    for (auto &item : queryDeviceLists) {
+        try {
+            std::vector<std::string> device_cap =
+                GetCore()->GetMetric(item, METRIC_KEY(OPTIMIZATION_CAPABILITIES));
+            for (auto &dc : device_cap) {
+                if (dc == METRIC_VALUE(BATCHED_BLOB)) {
```

Comment:
why do we need to ignore it?

Reply:
because when we use AUTO:CPU, but if the GPU is availble, we will get `BATCHED_BLOB` metric, which will cause `TEST_P(SetBlobOfKindTest, CompareWithRefs)` failed, so I think the `BATCHED_BLOB` is not supported for AUTO.
otherwise, maybe we should change the logic of current `OPTIMIZATION_CAPABILITIES`, should we make it equal to intersection for all devices?

Reply:
ok, I see your point. I think we can skip BATCHED_BLOB for now